cypress/support/pages/settings.username.pageObject.js

```diff
@@ -0,0 +1,9 @@
+class SettingsUsername {
+  updateUsername(newUsername) {
+    // eslint-disable-next-line cypress/unsafe-to-chain-command
+    cy.get('input[placeholder="Username"]').clear().type(newUsername);
```

Comment:
Consider using `data-cy` attributes for selecting elements in your tests as per the task requirements. This will make your tests more robust and less prone to breaking if the UI changes.